packages/@ember/-internals/glimmer/lib/helpers/concat.ts

```diff
@@ -22,10 +22,14 @@ const normalizeTextValue = (value: any): string => {
   Example:
 
   ```handlebars
-  {{some-component name=(concat firstName " " lastName)}}
+  <SomeComponent @name={{concat firstName " " lastName}} />
```

Comment:
You actually don't need `concat` in this example at all (this was one of the selling points of angle bracket invocation). This works perfectly:

```hbs
<SomeComponent @name="{{firstName}} {{lastName}}" />
```

---

Maybe we don't need dual examples in this case?